lib/raygun.ts

```diff
@@ -335,6 +352,31 @@ class Raygun {
     }
   }
 
+  /**
+   * Attach as express middleware to create a breadcrumb store scope per request.
+   * e.g. `app.use(raygun.expressHandlerBreadcrumbs);`
+   * Then call to `raygun.addBreadcrumb(...)` to add breadcrumbs to the future Raygun `send` call.
+   * @param req
+   * @param res
+   * @param next
+   */
+  expressHandlerBreadcrumbs(req: Request, res: Response, next: NextFunction) {
+    breadcrumbs.runWithBreadcrumbs(() => {
+      breadcrumbs.addRequestBreadcrumb(req, res);
+      // Make the current breadcrumb store available to the express error handler
+      res.locals.breadcrumbs = breadcrumbs.getBreadcrumbs();
```

Comment:
This allows to obtain the scoped breadcrumbs store in the `expressHandler` method, as described below.